### ISSUE TITLE
Duplicate Variable Compound Name values in Sept 20th snapshot

### ISSUE THREAD

**Issue by martinjuckes** [CMIP-Data-Request/Harmonised-Consultation-Phase-1#25]
There are 36 variables with duplicate compound names in the Sept 20th snapshot of Air Table: 
['6hrPlevPt.cldicemxrat', '6hrPlevPt.ta', 'Amon.ch4', 'Amon.co2', 'Amon.n2o', 'Amon.o3', 'CFmon.clmodisice', 'CFmon.clmodisliquid', 'E1hr.pr', 'E1hr.sfcWind', 'E1hr.ta', 'E1hr.ua', 'E1hr.va', 'E1hr.wap', 'E3hrPt.hus', 'Emon.hus', 'Emon.ua', 'Emon.va', 'Oclim.difmxybo', 'Oclim.difmxylo', 'Oclim.diftrbbo', 'Oclim.diftrblo', 'Oclim.diftrebo', 'Oclim.diftrelo', 'Oclim.diftrxybo', 'Oclim.diftrxylo', 'Oclim.dispkexyfo', 'Oclim.tnkebto', 'Oday.zos', 'Omon.ficeberg', 'Omon.hfibthermds', 'Omon.hfrunoffds', 'Omon.hfsifrazil', 'Omon.hfsnthermds', 'Omon.hfx', 'Omon.hfy']

Some, such as `hfx`, `hfx` are duplicate because there has been a request for the variable on a different grid. 

Others, e.g. E1hr.ta, are there because the distinction between the output name, `ta`, and the CMOR record label, `ta27` has been lost in the process of adapting to Air Table. 

In CMIP5 and CMIP6 there were some options indicated by having pairs of CMOR records with the same output name, generally with different choices for vertical resolution. 

In CMIP7 we want to indicate options with the conditional flag. Provisionally, we can also re-insert the CMIP7 names `ta27` etc in the disambiguation column.

- E1hr.pr: appears to be a mistaken submission
- E1hr.sfcWind: there is a new 100m version: should have a name change for consistency with ua100m.
- cldicemxrat : one should have been cldwatmxrat [Corrected]
- 6hrPlevPt.ta --> ta27
- Amon.ch4, co2, n2o, o3: clim and globalClim variations. For co2, n2o there is some confusion in naming which needs to be fixed.
-  'CFmon.clmodisice', 'CFmon.clmodisliquid' -- duplicate submissions.
- 'Oclim.difmxybo', 'Oclim.difmxylo', 'Oclim.diftrbbo', 'Oclim.diftrblo', 'Oclim.diftrebo', 'Oclim.diftrelo', 'Oclim.diftrxybo', 'Oclim.diftrxylo', 'Oclim.dispkexyfo', 'Oclim.tnkebto' : '2d' added.

- 'Omon.ficeberg' : 2d added.
- 
-  'Omon.hfibthermds', 'Omon.hfrunoffds', 'Omon.hfsifrazil', 'Omon.hfsnthermds' : 2d added. 

Oday.zos : duplicate submission.

**Comment by martinjuckes** [comment]
For duplications I've added "_dup" in the "Disambiguation" column.

**Comment by CMIP-IPO** [comment]
Just to flag the 'CFmon.clmodisice', 'CFmon.clmodisliquid' compound name duplicates are not duplicate variables - the differences are in the dimensions and require new grids. The proposer should likely submit different physical parameters to reflect these differences. I will go back to them on this next week - BD

**Comment by martinjuckes** [comment]
@CMIP-IPO Yes, he needs something like "MODIS Liquid-Topped-Cloud Water Path".
